### ISSUE TITLE
Missing TrackExtension  (not TrackPointExtension)

### ISSUE THREAD

**Issue by delp14** [astrowonk/gpxcsv#7]
Hi, I'm using a Garmin navigator.
gpx contains a TrackExtension, but the data is lost after running the program.

In my example, the colour of the track I want to save is used.

sample gpx:

```
<?xml version="1.0" encoding="utf-8"?><gpx creator="Garmin Desktop" version="1.1" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd 

  <metadata>
    <link href="http://www.garmin.com">
      <text>Garmin International</text>
    </link>
    <time>2022-01-24T07:00:22Z</time>
    <bounds maxlat="54.878299031406641" maxlon="58.571017971262336" minlat="54.823138983920217" minlon="58.462714981287718" />
  </metadata>

  <trk>
    <name>Distance traveled</name>
    <extensions>
      <gpxx:TrackExtension>
        <gpxx:DisplayColor>Black</gpxx:DisplayColor>
      </gpxx:TrackExtension>
    </extensions>
    <trkseg>
      <trkpt lat="54.878299031406641" lon="58.571017971262336">
        <ele>632.07000700000003</ele>
        <time>2021-01-24T10:38:34Z</time>
      </trkpt>
      <trkpt lat="54.878269024193287" lon="58.570830971002579">
        <ele>633.14001399999995</ele>
        <time>2021-01-24T10:38:43Z</time>
      </trkpt>
```

output:

```

[{'ele': 632.070007,
  'extensions': '\n      ',
  'lat': 54.87829903140664,
  'lon': 58.571017971262336,
  'name': 'Distance traveled',
  'time': '2021-01-24T10:38:34Z'},
 {'ele': 633.140014,
  'extensions': '\n      ',
  'lat': 54.87826902419329,
  'lon': 58.57083097100258,
  'name': 'Distance traveled',
  'time': '2021-01-24T10:38:43Z'},

```

**Comment by astrowonk** [comment]
Thanks for the bug report. Yeah I've never seen extension data encoded quite like this, usually I see:

```
<extensions>
					<gpxtpx:TrackPointExtension>
						<gpxtpx:hr>156</gpxtpx:hr>
						<gpxtpx:cad>71</gpxtpx:cad>
						<gpxtpx:speed>0.254569</gpxtpx:speed>
						<gpxtpx:course>289.910889</gpxtpx:course>
						<gpxtpx:hAcc>5.644999</gpxtpx:hAcc>
						<gpxtpx:vAcc>5.373536</gpxtpx:vAcc>
					</gpxtpx:TrackPointExtension>
				</extensions>

```

I'll see what I can do with this tonight / this weekend, should be possible to fix this quickly. It looks like trk level data so it'd be the same for the whole trk, which I think I already do with trk name.

**Comment by astrowonk** [comment]
@delp14 are you sending the entire header? The first line looks like it's missing some stuff, I'm not even able to convert with the missing columns per your original message.

**Comment by astrowonk** [comment]
Fixed via 9eb85ea8f44cb01d2f3bdb2dd9cf20a9b2219a4d.

Should be on pypi;  I messed up the unit test coverage which I fixed subsequently but I've already pushed [0.2.15](https://pypi.org/project/gpxcsv/0.2.15/) to pypi.
